### PR TITLE
MangoApps Guides Crawler

### PR DESCRIPTION
**Pull request by siddharthaatmango**
This will crawl all the MangoApps guides pages

### CODE REVIEW

**Review by shreyasmangoapp — APPROVED** [review]

**Review by shreyasmangoapp — APPROVED** [review]

**Review by shreyasmangoapp — APPROVED** [review]

**Review by shreyasmangoapp — APPROVED** [review]

File: loader_hub/mangoapps_guides/base.py [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        results = []
+
+        self.fetched_urls = self.crawl_urls()
+
+        self.guides_pages = {}
+        self.failed_urls = []
+
+        for url in self.fetched_urls:
+            scraper = GuidesScraper(url)
+            print(f"Url scraping: {url}")
```
Comment (jerryjliu) [inline loader_hub/mangoapps_guides/base.py:43]: can you remove print statements?

File: loader_hub/mangoapps_guides/base.py [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+class MangoppsGuidesReader(BaseReader):
+    """MangoppsGuides reader. Reads data from a MangoppsGuides workspace.
+
+    Args:
+        domain_url (str): MangoppsGuides domain url
+        limir (int): depth to crawl
+    """
+
+    def __init__(self, domain_url: str, limit: int) -> None:
```
Comment (jerryjliu) [inline loader_hub/mangoapps_guides/base.py:22]: seems like domain_url and limit can be parameters in load_data, not as `__init__` (i would think of initialization params as those that would be common to all requests)

File: loader_hub/mangoapps_guides/base.py (hunk 1 above)
Comment (siddharthaatmango) [reply loader_hub/mangoapps_guides/base.py]: done

File: loader_hub/mangoapps_guides/base.py (hunk 2 above)
Comment (siddharthaatmango) [reply loader_hub/mangoapps_guides/base.py]: thanks, it is done now

**Review by jerryjliu — APPROVED** [review]

**Review by shreyasmangoapp — APPROVED** [review]

**Pull request merged**